components/admission-webhook/manifests/base/deployment.yaml

```diff
@@ -18,6 +18,15 @@ spec:
         ports:
         - name: https-webhook
           containerPort: 4443
+        securityContext:
+          runAsNonRoot: true
+          allowPrivilegeEscalation: false
+          runAsUser: 1000
```

Comment:
Is there any particular reason that `UID: 1000` was chosen, I think its not correct?

The Docker image should run as `65532` in most cases:
- https://github.com/kubeflow/kubeflow/blob/v1.10.0/components/admission-webhook/Dockerfile#L22

Also, perhaps we should not specify a `runAsUser` in the first place, as we already have `runAsNonRoot`.

Reply:
So far 1000 works in our tests. I mean we can of course change it, but having a proper securitycontext first is a massive improvement and then in a follow up PR we can do refinements.